### PR TITLE
Update secured asset transfer to use deployCC script

### PR DESCRIPTION
**Pull request by nikhil550**
Signed-off-by: NIKHIL E GUPTA <negupta@us.ibm.com>

#### Type of change

- Documentation update

#### Description

Updated the secured asset tutorial to deploy the smart contract using the deployCC script, now that the capability is there. This will shorten the tutorial for new users.

### CODE REVIEW

File: docs/source/secured_private_asset_transfer_tutorial.md [hunk 1]
```diff
@@ -67,22 +67,35 @@ After the two organizations have agreed to the same price, the asset owner can u
...
-
+We are going to use the Fabric test network to run the secured asset transfer smart contract. Open a command terminal and navigate to test network directory in your local clone of [fabric-samples](https://github.com/hyperledger/fabric-samples). We will operate from the `test-network` directory for the remainder of the tutorial.
 ```
 cd fabric-samples/test-network
 ```
 
-You can then deploy the network with the following command:
+First, bring down any running instances of the test network:
+```
+./network down
```
Comment (denali49) [inline docs/source/secured_private_asset_transfer_tutorial.md]: ./network.sh down

**Review by denali49 — CHANGES_REQUESTED** [review]
Thanks Nik, see inline comment, need to correct ./network down to ./network.sh down
Other than that, all good.

File: docs/source/secured_private_asset_transfer_tutorial.md (hunk 1 above)
Comment (nikhil550) [reply docs/source/secured_private_asset_transfer_tutorial.md]: Got catch. done.

**Review by denali49 — APPROVED** [review]

**Review by denali49 — APPROVED** [review]
lgtm, thanks Nik.

**Pull request merged**